Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM broadinstitute/gatk:gatkbase-1.2
 ADD . /gatk
 
 WORKDIR /gatk
-RUN /gatk/gradlew clean compileTestJava installAll localJar
+RUN /gatk/gradlew clean compileTestJava installAll localJar -Drelease=true
```

Comment:
Can you check whether the `build_docker.sh` script uses this `Dockerfile`? If not, you'll need to modify that as well.

Reply:
I have tested using the build_docker.sh. It does indeed use this `Dockerfile` as that is how I generated the above command line output.

Reply:
In that case, go ahead and merge when ready